libgobuster/state.go

```diff
@@ -36,39 +36,42 @@ type StringSet struct {
 // Contains State that are read in from the command
 // line when the program is invoked.
 type State struct {
-	Client         *http.Client
```

Comment:
Not sure why this is all marked as such a big block of changes. Did the whitespacing used change or something? Does this match `gofmt`?

Reply:
This does match gofmt, but I have no idea either.

Reply:
Haha, fair enough. Not a big issues